src/pages/Home.vue

```diff
@@ -112,10 +135,13 @@ const events = {
     itemStore.addItem({ text, slot }, props.podcastId, docname)
   },
   onUpdateSaveDoc() {
+    console.log(props.podcastId,"props.podcastId")
     itemStore.saveData(props.podcastId, docname)
   },
   onClickDelete(item: Item) {
-    itemStore.removeItem(item, props.podcastId, docname)
+    if (window.confirm("are you sure?")) {
```

Comment:
This pull request has nothing to do with this change. Please keep the scope of each pull request contained